src/rules/__tests__/no-standalone-expect-test.ts

```diff
@@ -0,0 +1,44 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import rule from '../no-standalone-expect';
+
+const ruleTester = new TSESLint.RuleTester({
+  parserOptions: {
+    ecmaVersion: 2015,
+    sourceType: 'module',
```

Comment:
do we need this default sourcetype?

Reply:
We do not, I definitely copied this from someone else... oops.